llvm/include/llvm/Support/Format.h

```diff
@@ -236,8 +236,7 @@ class FormattedBytes {
       : Bytes(B), FirstByteOffset(O), IndentLevel(IL), NumPerLine(NPL),
         ByteGroupSize(BGS), Upper(U), ASCII(A) {
 
-    if (ByteGroupSize > NumPerLine)
-      ByteGroupSize = NumPerLine;
+    ByteGroupSize = std::min<uint32_t>(ByteGroupSize, NumPerLine);
```

Comment:
IIRC, the best practice is to cast arguments instead of specifying template arguments (since I'm not sure if the exact number and order of template arguments is guaranteed to stay the same in the future versions of C++).